packages/components/pages/f-loyalty/src/components/Loyalty.vue

```diff
@@ -1,19 +1,22 @@
 <template>
     <div role="main">
         <div
-            v-if="!isAppsUserAgent"
+            v-if="!isAppsUserAgent && !isInIframe"
```

Comment:
Would prefer to unify these both into a single computed property named something like `showLoyaltyNavigationElements` to help with readers' understanding here.